.github/workflows/deploy-prod.yml

```diff
@@ -1,27 +0,0 @@
-name: Deploy Widgets to Mainnet
-on:
-  push:
-    branches: [main]
-jobs:
-  deploy-widgets:
-    runs-on: ubuntu-latest
-    name: Deploy widgets to social.near (mainnet)
-    env:
-      NEAR_SOCIAL_DEPLOY_ID: ${{ vars.NEAR_SOCIAL_DEPLOY_ID }}
-      NEAR_SOCIAL_ACCOUNT_ID: ${{ vars.NEAR_SOCIAL_ACCOUNT_ID }}
-      NEAR_SOCIAL_ACCOUNT_PUBLIC_KEY: ${{ vars.NEAR_SOCIAL_ACCOUNT_PUBLIC_KEY }}
-      NEAR_SOCIAL_ACCOUNT_PRIVATE_KEY: ${{ secrets.NEAR_SOCIAL_ACCOUNT_PRIVATE_KEY }}
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Install near-social CLI
-        run: |
-          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/FroVolod/near-social/releases/download/v0.2.5/near-social-v0.2.5-installer.sh | sh
-
-      - name: Deploy widgets
-        run: |
-          which near-social
-          echo $PATH
-          ~/.cargo/bin/near-social deploy "$NEAR_SOCIAL_DEPLOY_ID" sign-as "$NEAR_SOCIAL_ACCOUNT_ID" network-config mainnet sign-with-plaintext-private-key --signer-public-key "$NEAR_SOCIAL_ACCOUNT_PUBLIC_KEY" --signer-private-key "$NEAR_SOCIAL_ACCOUNT_PRIVATE_KEY" send
```

Comment:
@milenakrawczyk the new prod deploy workflow does not match the logic that was here. We need to deploy to `NEAR_SOCIAL_DEPLOY_ID` and sign as `NEAR_SOCIAL_ACCOUNT_ID`

Reply:
@mpeterdev someone merged something incorrectly. I fixed it here: https://github.com/near/near-discovery-components/pull/110/commits/7c23bc63d7941b136b69061059d1c8ba24f74398. We now have too many workflow files: 
<img width="341" alt="image" src="https://github.com/near/near-discovery-components/assets/66628214/defea395-49fb-4a3b-b521-7d10fd230a95">

Reply:
I suggest to delete deploy-prod.yml and deploy-dev.yml

Reply:
Good catch, thanks! I fixed it in this PR: https://github.com/near/near-discovery-components/pull/130